cleantalk.antispam/include.php

```diff
@@ -1141,11 +1105,9 @@ function ctKeyStopStopListening(){
                     }
 
                     if(typeof window.addEventListener == 'function'){
-                        window.addEventListener('mousemove', ctFunctionMouseMove);
                         window.addEventListener('mousedown', ctFunctionFirstKey);
                         window.addEventListener('keydown', ctFunctionFirstKey);
                     }else{
-                        window.attachEvent('onmousemove', ctFunctionMouseMove);
```

Comment:
Why the mousemove collection was removed ? I can't see this in another place.

Reply:
https://doboard.com/task/8738#comment_57189